### ISSUE TITLE
[BUG] Can't Install .jar files and the game crashes on 1.19+ versions

### ISSUE THREAD

**Issue by Flysakib** [PojavLauncherTeam/PojavLauncher#5271]
### Describe the bug

I can't install Any .jar file and also i can't play v1.19+ version. When i try to run those versions. The app crashes and says "application error with code 1 check latest log for more info" . and again the holy fast renderer glitches . i cant see some of the world chunks.

### The log file and images/videos

[latest.log](https://github.com/PojavLauncherTeam/PojavLauncher/files/14807822/latest.log)

### Steps To Reproduce

```markdown
??
```

### Expected Behavior

It say application exited with code 1 check latest log for more info

### Platform

```markdown
- Device model: Samsung galaxy m21
- CPU architecture: 4x ARM Cortex-A73 @ 2.31GHz
4x ARM Cortex-A53 @ 1.74GHz
- Android version: 12
- PojavLauncher version: edelweiss-20230928-9484d0c-v3_openjdk
```

### Anything else?

I can't istall any type of .jar file , cant run v1.19+ versions , cant use holy fast renderer ( because i can't see some of the world chunks )

**Comment by GaucovyAero** [comment]
send latestlog.txt

**Comment by Flysakib** [comment]
Here it is!

**Comment by Flysakib** [comment]
[latest.log](https://github.com/PojavLauncherTeam/PojavLauncher/files/14807829/latest.log)

**Comment by GaucovyAero** [comment]
> [latest.log](https://github.com/PojavLauncherTeam/PojavLauncher/files/14807829/latest.log)

thats not latestlog.txt, thats latest.log

**Comment by GaucovyAero** [comment]
the location of latestlog.txt is in the issue template

**Comment by GaucovyAero** [comment]
stale issue

**Comment by Flysakib** [comment]
i got the latestlog.txt 
[latestlog.txt](https://github.com/PojavLauncherTeam/PojavLauncher/files/14872779/latestlog.txt)

I m so sorry i was offline for 1 week. By the way pls fix this issue i only be able to play in v1.16.5 and below versions and still cant install .jar files . pls dont close my bug report . plz fix this issue as soon as possible
